packages/web/src/containers/earn-add-liquidity-container/EarnAddLiquidityContainer.tsx

```diff
@@ -829,6 +829,13 @@ const EarnAddLiquidityContainer: React.FC = () => {
     return isFetchingPools || isLoadingCommon;
   }, [isFetchingPools, isLoadingCommon]);
 
+  const showOneClickStaking = useMemo(
+    () =>
+      selectPool.poolPath ===
+      "gno.land/r/demo/wugnot:gno.land/r/gnoswap/gns:3000",
```

Comment:
I mean `poolPath` is hardcoded here. and this code only affect to `gno.land/r/demo/wugnot:gno.land/r/gnoswap/gns:3000`

Report to Jaeho if you don't have way to get data about whether pool is stakable